.github/workflows/dev-test.yaml

```diff
@@ -29,6 +29,9 @@ jobs:
           - "tests/e2e/settings"
           - "tests/e2e/basic_widgets"
           - "tests/e2e/plate_based_widgets"
+          # - "tests/e2e/heatmap"
+          # - "tests/e2e/stimulation"
+
```

Comment:
the stim test suite isn’t running yet and the heatmap tests are all failing, but I will once I double check the dimensions in the desktop app because any px difference will cause them all to fail. But they're set up